### PR TITLE
Review PR for Bxmsdoc 5789 Managing assets in Business Central

### PR DESCRIPTION
**Pull request by michelehaglund**
Updated asset list and screenshot.

Original jira: https://issues.redhat.com/browse/BXMSDOC-5789
Rendered output: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-5789/#assets_types_ref

### CODE REVIEW

File: doc-content/enterprise-only/getting-started/assets-creating-proc.adoc [hunk 1]
```diff
@@ -13,11 +13,4 @@ NOTE: Migrating business processes is an irreversible process.
...
 
-NOTE: If you have not created a project, you can either add a project, use a sample project, or import an existing project.
-
-.Add a new project
-Click *Add Project* and input the new project name.
-
-.Use a sample project
-.. Click the arrow next to *Add Project* and select *Try Samples*.
-.. Select a project and click *Ok*.
+NOTE: If you have not created a project, you can either add a project, use a sample project, or import an existing project. See {URL_MANAGING_PROJECTS}[_{MANAGING_PROJECTS}_].
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/assets-creating-proc.adoc:16]: For more information, see

File: doc-content/enterprise-only/getting-started/assets-types-ref.adoc [hunk 2]
```diff
@@ -19,13 +19,13 @@ Business Processes are diagrams that describe the steps necessary to achieve bus
...
 +
 IMPORTANT: The business process application example includes features that are Technology Preview only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and are not recommended for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
-For more information about Red Hat Technology Preview support, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+//For more information about Red Hat Technology Preview support, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 
-* Case definition (legacy)
+* Case Definition
```
Comment (emmurphy1) [inline doc-content/enterprise-only/getting-started/assets-types-ref.adoc:24]: Case definition

I don't see any reason to cap. Goes for the who list. Data object. Decision table. Global variables. No (). The singular is implied in the plural.

Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/assets-types-ref.adoc:24]: This is the UI usage. I will request a change.

File: doc-content/enterprise-only/getting-started/assets-creating-proc.adoc (hunk 1 above)
Comment (michelehaglund) [reply doc-content/enterprise-only/getting-started/assets-creating-proc.adoc:16]: Done :)

**Pull request merged**